ci/code_checks.sh

```diff
@@ -197,6 +197,11 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -r -E --include '*.py' --exclude testing.py '(numpy|np)(\.testing|\.array_equal)' pandas/tests/
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    # GH37053, Check for np.random.<method> instead of `from numpy.random import` or `from numpy import random`
```

Comment:
this can now be done in the pre-commit checks similar to: https://github.com/pandas-dev/pandas/blob/master/.pre-commit-config.yaml#L75

Reply:
done